app/components/ledger/CredentialInfoLedgerDetails.tsx

```diff
@@ -23,21 +22,18 @@ const CredentialInfoLedgerDetails = (props: Props) => (
         <p>
             <b>Signature threshold:</b> {props.credentialPublicKeys.threshold}
         </p>
+        <div>
+            <b>Anonymity Revocation threshold:</b>
+            <p className="mT0">
+                {props.revocationThreshold} out of{' '}
+                {Object.keys(props.arData).length}
+            </p>
+        </div>
         <p>
-            <b>Registration ID credential (RegIdCred):</b>
-            <DisplayHexString className="mV40" value={props.credId} />
+            <b>Identity Valid to:</b> {formatDate(props.policy.validTo)}
```

Comment:
It is not the identity that will be valid to that point in time but the credential.

Reply:
hmm

Reply:
I just think it should say Valid to, with no prefix. If not, then someone else needs to decide what they want for this.

Reply:
I will delete this.